code/modules/clothing/glasses/hud.dm

```diff
@@ -180,3 +180,91 @@
 
 /obj/item/clothing/glasses/hud/health/tajblind/attack_self()
 	toggle_veil()
+
+/obj/item/clothing/glasses/hud/modular
+	name = "modular hud attachment device"
+	desc = "A modular hud attachment device that has two slots to secure multiple HUDs."
+	icon_state = "mhad_0"
+	item_state = "mhad"
+	species_fit = list("Vox") //Random pasta from med/sec huds. Ionward made sprites for all these species, so it's not pointless pasta.
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/eyes.dmi',
+		"Drask" = 'icons/mob/species/drask/eyes.dmi',
+		"Grey" = 'icons/mob/species/grey/eyes.dmi'
+		)												// End random pasta
+	var/huds = 0
+	materials = list(MAT_METAL = 5000, MAT_GLASS = 1000)
+
+/obj/item/clothing/glasses/hud/modular/attack_hand(mob/user as mob, pickupfireoverride = FALSE)
+	if(loc != user)
+		. = ..()
+	else if (huds)
+		for(var/obj/item/clothing/glasses/hud/h in src)
+			h.remove_hud(user, src)
+			huds --
+			user.put_in_active_hand(h)
+			break
+		update_icon()
+
+/obj/item/clothing/glasses/hud/modular/attackby(obj/item/I, mob/living/carbon/human/user, params)
+	if(istype(I, /obj/item/clothing/glasses/hud) && !istype(I, src.type) && istype(user))
+		var/obj/item/clothing/glasses/hud/hud = I
+		if(huds < 2)
+			huds ++
+			user.drop_item()
+			I.loc = src
+			if(hud.invis_view == SEE_INVISIBLE_MINIMUM)
+				invis_view = SEE_INVISIBLE_MINIMUM
+				update_invis()

```

Comment:
why are you calling this.

Reply:
god damn it forgot to remove that, its from a past branch that somehow snuck its way into this one.

Reply:
wait nver mjknd